### DISCUSSION TITLE
M1 chip running docker in centos8 virtual machine running pulsar error

### DISCUSSION THREAD

**Discussion by spiritdxl** [apache/pulsar#19950] [Q&A] [answered]
Running PD virtual machine on mac with M1 chip and running Centos8-steam, docker running pulsar fails, but running pulsar locally with docker has no problem. Linux is also an ARM version
Error report figure
<img width="753" alt="image" src="https://user-images.githubusercontent.com/129169540/228207071-0aa45d51-5aae-4f95-bb0d-542a7a1bc91e.png">
<img width="561" alt="image" src="https://user-images.githubusercontent.com/129169540/228207243-e43b01c3-3b5a-4960-bdf6-ea9d8ea4703c.png">

M1 starts successfully as follows
<img width="956" alt="image" src="https://user-images.githubusercontent.com/129169540/228207416-bf229e52-4a45-40c6-80c5-4ed0d4c64279.png">

The following command is used
docker run -it -e PULSAR_PREFIX_forceDeleteNamespaceAllowed=true -p 6650:6650  -p 8080:8080 --mount source=pulsardata,target=/pulsar/data --mount source=pulsarconf,target=/pulsar/conf apachepulsar/pulsar:2.11.0 sh -c "bin/apply-config-from-env.py conf/standalone.conf && bin/pulsar standalone"

**Comment by tisonkun** [comment, marked as answer]
Hi @spiritdxl! Pulsar has not currently released Docker image for ARM64 platform.

You can package by yourself, which is supported by https://github.com/apache/pulsar/pull/17733.

The community is currently working on integrating ARM64 Docker image support in the release process, which you can subscribe to updates - https://github.com/apache/pulsar/pull/19432

The overall issue is here https://github.com/apache/pulsar/issues/12944.
